pkg/config/config.go

```diff
@@ -567,6 +567,9 @@ type NetworkConfig struct {
 	// NetavarkPluginDirs is a list of directories which contain netavark plugins.
 	NetavarkPluginDirs attributedstring.Slice `toml:"netavark_plugin_dirs,omitempty"`
 
+	// DefaultFirewall is the default firewall
+	DefaultFirewall string `toml:"default_firewall,omitempty"`
```

Comment:
Default firewall makes little sense as name, it should just be `firewall_driver` similar to network_backend. Please rename this in all cases.